.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -43,8 +37,8 @@ body:
     attributes:
       label: "Steps to reproduce"
       description: |
-        Steps to reproduce using `nvim -u NORC` and/or `nvim -u NONE` (please test both).
-        If you are reporting build failures, please list the exact sequence of steps including all CMake flags (if any).
+        If you cannot reproduce the issue using `nvim -u NORC`, prepare and upload a [minimal config](https://github.com/neovim/neovim/blob/master/contrib/minimal.lua) that reproduces the issue with `nvim -u minimal.lua`.
```

Comment:
Upload where? In a gist or paste it in the report?

Reply:
I'm open for suggestions how best to share the customized template :)

Reply:
> Good question; I flipped-flopped on that as well, hence it's not mentioned :) Unfortunately, `.lua` is not a supported filetype for uploading...

For uploading where? In gists?

Reply:
By attaching via drag & drop to the issue itself.

Reply:
Yes, there's only a (somewhat eclectic) list of allowed filetypes.

I don't like the spoiler tag; if it's truly minimal, just copying the _added_ lines in the "steps" field would work.